code/modules/recycling/disposal/pipe.dm

```diff
@@ -97,7 +97,10 @@
 		floorturf = T
 		if(floorturf.floor_tile)
 			new floorturf.floor_tile(T)
-		floorturf.make_plating()
+		if(istype(floorturf, /turf/open/floor/engine))
+			floorturf.make_plating(TRUE)
+		else
+			floorturf.make_plating()
```

Comment:
is there any case where you wouldn't want the tile to break?

Reply:
yes, space tiles and open space tiles are technically considered floors. and those 2, as well as reinforce floor, is the only floors which have make_plating be false. obviously we dont want to run make plating on space or openspace, only reinforced floor, so we dont want to force it for every floor tile

Reply:
It feels like a situation that could apply elsewhere. Maybe consider handling it in a new `/turf/open/floor` proc?

Reply:
Or a new arg to `make_plating`

Reply:
ill try

Reply:
made a proc

Reply:
> 
> 
> yes, space tiles and open space tiles are technically considered floors. and those 2, as well as reinforce floor, is the only floors which have make_plating be false. obviously we dont want to run make plating on space or openspace, only reinforced floor, so we dont want to force it for every floor tile

Are you sure about that? make_plating is defined on /turf/open/floor, and space tiles are /turf/open/space.
We only care about force on reinforced floors, since they're the only thing that overrides make_plating outside of platings. Wouldn't just passing in true be acceptable here?

Reply:
my bad, not space, i was tired when i said that, i meant plating, plating is considered a floor tile and we dont want that to have make_plating called for it, glass floors also dont want that called for it, as those function visually like plating as well @LemonInTheDark

Reply:
also, i wanted to future proof it in case theres anything we dont want to make plating but count as a floor tile, i can think of a few examples, such as a liquidy tile where you just fly through the tile without breaking it, or a non-physical tile. idk, this way is future proofed and i believe thats going to be better in the long run

Reply:
I can get behind adding some way of preventing floor_tile creation, but wouldn't that be better served by a proc? there are multiple bits of the code that do something similar to this, so a more general solution would be better.

As for make_plating, if something really just doesn't want to be made into plating, it can override the proc in the same way `/turf/open/floor/plating` does. I suppose force is rather poorly named, as it is only used by `/turf/open/floor/engine/make_plating()`, it was added to solve a similar issue with singularity pulling a couple months back.

Reply:
i dont think a proc to prevent floor tile creation is the best solution here since i dont want it to be prevented on a tiles subtype inherently, only in instances in which a tile is set to not break, this can occur in multiple times and not just with the engine tile. for example: indestructible versions of tiles wont break, so every indestructible subtype needs to be able to not pop if its not forced to break

Reply:
Wouldn't it be better to make a proc that spawns floor tiles, then replace everything that spawns floor tiles with the proc, then override the proc to nothing if something doesn't want to spawn floor tiles? This all ignores that things that don't want to spawn floor tiles can just set their floor tile to null and it'll work just fine.